va/x11/va_dri3.c

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright © 2022 Collabora Ltd.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
```

Comment:
I would propose to use SPDX instead of whole license text

Reply:
AFAICT no SPDX notations are used in the project. So while I'm in favour (esp with it's an ISO standard), it's something better handled a) as follow-up and b) across the whole project.